src/Paket.Core/Dependencies/NuGetV3.fs

```diff
@@ -109,9 +109,6 @@ let internal findAutoCompleteVersionsForPackage(v3Url, auth, packageName:Domain.
     async {
         let url = sprintf "%s?semVerLevel=2.0.0&id=%O&take=%d%s" v3Url packageName (max maxResults 100000) (if includingPrereleases then "&prerelease=true" else "")
 
-        if verbose then
-            verbosefn "findAutoCompleteVersionsForPackage from url '%s'" url
-
```

Comment:
Because in this particular instance all these logging statements missed the critical one, therefore I moved the verbose logging down to the `Utils.fs` to track all requests (including future ones).

After the logging change you could see that it wasn't actually a deadlock but just paket doing an endless loop of requests with the server (but the requests were not logged anywhere)